packages/blade/src/components/StepGroup/_decisions/decisions.md

```diff
@@ -0,0 +1,228 @@
+| title        | Title of StepItem                                                                                 | string                                                             |                                       |
+| timestamp    | A string that renders in italic font. Made for adding timestamp values                            | string                                                             |                                       |
+| description  | Description of StepItem                                                                           | string                                                             |                                       |
+| stepProgress | Progress line of step. When its `start` only starting part is highlighted and rest is kept dotted | 'start' \| 'end' \| 'full' \| 'none'                               | 'none'                                |
+| leading      | Leading element                                                                                   | ReactElement (StepItemIcon or StepItemIndicator components as JSX) | <StepItemIndicator color="neutral" /> |
+| trailing     | Trailing element                                                                                  | ReactElement (Badge components as JSX)                             | undefined                             |
+| isSelected   | Selected state of item                                                                            | boolean                                                            | undefined                             |
+| href         | Anchor tag's href value. Turns StepItem into interactive item and render it as anchor             | string                                                             | undefined                             |
+| onClick      | StepItem's click handler. Turns StepItem into interactive item and render it as button tag        | () => void                                                         | undefined                             |
+| children     | Slot                                                                                              | ReactNode                                                          | undefined                             |
+
+### StepItemIcon
+
+| **Props** | **Description** | **Type**                                                                        | **Default Value** |
+| --------- | --------------- | ------------------------------------------------------------------------------- | ----------------- |
+| icon      | icon component  | IconComponent                                                                   |                   |
+| color     | color of icon   | 'positive' \| 'negative' \| 'neutral' \| 'notice' \| 'information' \| 'primary' | 'neutral'         |
```

Comment:
Default isn't `primary`?

Reply:
nope. Primary is blue color so having it as default will make it seem like everything is important